src/components/Coaching/CoachingDetail/Activity/Activity.tsx

```diff
@@ -723,6 +474,57 @@ export const Activity: React.FC<ActivityProps> = ({
             </>
           )}
         </ActivitySection>
+        <ActivitySection data-testid="CurrentRealityPartnerFinancial">
+          {loading ? (
+            <MultilineSkeleton lines={1} width="90%" height="90%" />
```

Comment:
I think we can just use a regular skeleton here and the two places below.

```suggestion
            <Skeleton width="90%" height="90%" />
```